src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int AMOUNT = 10;
+    private static final FigureSupplier FIGURE_SUPPLIER = new FigureSupplier();
```

Comment:
make it not-static
```suggestion
    private final FigureSupplier figureSupplier= new FigureSupplier();
```

Reply:
The problem is non-static field can't be accessed from static main method.
So I either leave it as is, or create an instance of FigureSupplier class inside main method.
Or prove me wrong, please.